.github/workflows/linters.yml

```
@@ -186,6 +186,7 @@ jobs:
         with:
           persist-credentials: false
       - name: Get team members if possible
+        if: ${{ ((github.event.pull_request && github.event.pull_request.base.ref) || github.event.ref) == github.event.pull_request.base.repo.default_branch }}
```

```suggestion
        if: ${{ (github.event.pull_request.base.ref) || github.event.ref) == (github.event.pull_request.base.repo.default_branch || github.event.repository.default_branch) }}
```

You're currently checking if it's a PR on only one side of the expression, so it'll still not be valid when the trigger is not a PR.